test/example.rb

```diff
@@ -2,6 +2,12 @@ def greeting(name)
   puts "Hello, #{name}! Welocome to the Ruby typos test."
   puts 'languege' # typo
   puts 'knowlege' # typo
+  puts 'knowlege' # typo
```

Comment:
```suggestion
  puts 'knowledge' # typo
```
Should replace `knowlege` with `knowledge`.